src/app/(drawerPages)/art-reveal/page.tsx

```diff
@@ -72,6 +63,31 @@ export default function ArtReveal() {
 
   const { data: treasureBox, isLoading } = useTreasureBox({ gameId: GAME_ID });
 
+  const handleCTAPress = useCallback(
+    debounce(() => {
+      attackBox.mutate({
+        gameId: GAME_ID,
+        user: {
+          address: address!,
+          cbId: userPublicProfile?.subdomainProfile?.name,
+          ensName: userPublicProfile?.ensDomainProfile?.name,
+        },
+      });
+    }, 500),
+    [
+      address,
+      // attackBox,
```

Comment:
did this caused infinite rendering?

Reply:
Actually no, but we weren't handling spam clicks